locations/method_locations.go

```diff
@@ -48,3 +48,8 @@ func MethodOperationInfo(m *desc.MethodDescriptor) *dpb.SourceCodeInfo_Location
 func MethodSignature(m *desc.MethodDescriptor, index int) *dpb.SourceCodeInfo_Location {
 	return pathLocation(m, 4, int(apb.E_MethodSignature.TypeDescriptor().Number()), index) // MethodDescriptor.options == 4
```

Comment:
Could you also change the functions above to use `MethodOptions`? For example:

```suggestion
	return MethodOption(int(apb.E_MethodSignature.TypeDescriptor().Number()), index)
```